CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 ## v1.3.0 [unreleased]
 
+### Release Notes
+
+#### Continuous Query Statistics
+
+When enabled, each time a continuous query is completed, a number of details regarding the execution are written to the `cq_query` measurement of the internal monitor database (`_internal` by default). The tags and fields of interest are
```

Comment:
We typically just enable all stats for simplicity, but maybe it would be better to make them selectively enabled (or disabled).